grails-app/conf/Config.groovy

```diff
@@ -66,10 +66,10 @@ grails.exceptionresolver.params.exclude = ['password']
 grails.converters.json.circular.reference.behaviour = "INSERT_NULL"
 
 // Mail plugin config
-grails.mail.adminEmailAddress = "aatams_admin@emii.org.au"
-grails.mail.systemEmailAddress = "aatams_system@emii.org.au"
-grails.mail.supportEmailAddress = "info@emii.org.au"
-grails.mail.dataCommitteeEmailAddress = "aatams_data_committee@emii.org.au"
+grails.mail.adminEmailAddress = "aatams_admin@aodn.org.au"
+grails.mail.systemEmailAddress = "aatams_system@aodn.org.au"
+grails.mail.supportEmailAddress = "info@aodn.org.au"
+grails.mail.dataCommitteeEmailAddress = "aatams_data_committee@aodn.org.au"
```

Comment:
@kereid, note aatams email addresses which are being updated to point to aodn.org.au instead of emii.org.au.  These need to be setup before these changes are pushed through to production if they haven't bee already.